HISTORY.md

```diff
@@ -1,5 +1,8 @@
 # Rocksdb Change Log
 ## Unreleased
+### Bug Fixes
+* Fix wrong result being read from ingested file, when a key in the file happen to be prefix of another key also in the file. The issue can further cause more data corruption. The issue exists with rocksdb >= 5.0.0 since DB::IngestExternalFile() was introduced.
```

Comment:
I think this implies the problem happens every time a key is a prefix of another key. But in my understanding it requires the shared prefix extends multiple bytes into the footer of one of the keys (the first byte is the value type which isn't manipulated by the call to `UpdateInternalKey()`). Maybe clarify it only happens under certain conditions?

Reply:
updated.